views/container-statuses.html

```diff
@@ -0,0 +1,17 @@
+<div ng-if="containerStatuses.length == 0"><em>none</em></div>
+<dl ng-repeat="containerStatus in containerStatuses | orderBy:'name'" class="dl-horizontal">
```

Comment:
@liggitt It looks OK. I added more containers and container statuses to the fixture.